Client/Application/LeanplumIntegration.swift

```diff
@@ -132,23 +132,25 @@ class LeanplumIntegration {
 
         Leanplum.start(userAttributes: userAttributesDict)
 
-        track(eventName: LeanplumEventName.openedApp)
-
-        // Only drops Leanplum event when a user has installed Focus (from a fresh state or a re-install)
-        if profile?.prefs.boolForKey(PrefsKeys.HasFocusInstalled) == canInstallFocus() {
-            profile?.prefs.setBool(!canInstallFocus(), forKey: PrefsKeys.HasFocusInstalled)
-            if !canInstallFocus() {
-                track(eventName: LeanplumEventName.downloadedFocus)
+        Leanplum.onStartResponse({ _ in
```

Comment:
It looks like this gets executed on the main thread:

https://github.com/Leanplum/Leanplum-iOS-SDK/blob/master/Leanplum-SDK/Classes/Leanplum.m#L733-L740

I think this should be ok. We're getting the `profile`, but we're not explicitly fetching anything from the DB, just prefs. I do agree that we should check `.shouldSendToLP()` around this though.